packages/core/crates/core-strategy/src/strategy.rs

```diff
@@ -221,25 +221,23 @@ impl MultiStrategy {
         for strategy in cfg.strategies.iter() {
```

Comment:
Why not `into_iter()`?

Reply:
The `cfg.strategies` cannot be moved, because `cfg` is used in the return value.